packages/gafl-webapp-service/src/pages/licence-details/licence-length/__tests__/route.spec.js

```diff
@@ -30,5 +33,32 @@ describe('licence-length > route', () => {
       const result = await getData(mockRequest)
       expect(result.isLicenceForYou).toBeFalsy()
     })
+
+    it('licenceTypeDisplay is called with the expected arguments', async () => {
+      const catalog = Symbol('mock catalog')
+      const permission = Symbol('mock permission')
```

Comment:
These don't really need to be Symbol objects, that's more for testing the return value, but don't worry about changing it.